Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1,28 +1,51 @@
-import demistomock as demisto
-from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
-from CommonServerUserPython import *  # noqa
+from typing import Dict, Tuple, Callable
 
 import requests
-import traceback
-from typing import Dict, Any, Tuple, Callable
 
-requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
 
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
 ''' CONSTANTS '''
 
 DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
 
 API_SUFFIX = '/api/2.0/fo/'
 
 # Arguments that need to be parsed as dates
-DATE_ARGUMENTS = [
-    'launched_after_datetime', 'launched_before_datetime',
-    'expires_before_datetime', 'no_vm_scan_since',
-    'vm_scan_since', 'no_compliance_scan_since', 'last_modified_after',
-    'last_modified_before', 'last_modified_by_user_after', 'last_modified_by_user_before',
-    'last_modified_by_service_after', 'last_modified_by_service_before', 'published_after',
-    'published_before'
-]
+DATE_ARGUMENTS = {'launched_after_datetime': '%Y-%m-%d', 'launched_before_datetime': '%Y-%m-%d',
+                  'expires_before_datetime': '%Y-%m-%d', 'no_vm_scan_since': '%Y-%m-%d', 'vm_scan_since': '%Y-%m-%d',
+                  'no_compliance_scan_since': '%Y-%m-%d', 'last_modified_after': '%Y-%m-%d',
+                  'last_modified_before': '%Y-%m-%d', 'last_modified_by_user_after': '%Y-%m-%d',
+                  'last_modified_by_user_before': '%Y-%m-%d', 'last_modified_by_service_after': '%Y-%m-%d',
+                  'last_modified_by_service_before': '%Y-%m-%d', 'published_after': '%Y-%m-%d',
+                  'published_before': '%Y-%m-%d', 'start_date': '%m/%d/%Y'
+                  }
+
+
+# Markdown Builders
+def build_host_detection_table_to_markdown(parsed_outputs: List[Any]) -> str:
```

Comment:
I cant move it down, because the dict below it is using the function name. if we indeed build a new output builder for the host detection i can just delete it